Stepper: Add Form Integration How To topic (#7464)
DevExpress/devextreme-documentation: concepts/05 UI Components/Stepper/30 Implement a Wizard/05 Set Up App Layout.md

> To get started, add Stepper and MultiView to your project. Create a navigation panel by adding a `<div>` element for the active step caption and two [Button](/Documentation/Guide/UI_Components/Button/Overview/) components to a container:

Reviewer: ```suggestion
To get started, add a Stepper and a MultiView to your project. Create a navigation panel - a container with child items. Child items are a `<div>` element that displays the active step caption and two [Button](/Documentation/Guide/UI_Components/Button/Overview/) components:
```

> To get started, add Stepper and MultiView to your project. Create a navigation panel by adding a `<div>` element for the active step indicator and two [Button](/Documentation/Guide/UI_Components/Button/Overview/) components to a container:

Reviewer 2: in docs we use word 'indicator' for step item, usage this wird with another meaning may confuse customers https://js.devexpress.com/jQuery/Documentation/25_1/Guide/UI_Components/Stepper/Overview/#Elements

it is just a caption or smth like this

> To get started, add a Stepper and a MultiView to your project. Create a navigation panel - a container with child items. Child items are a `<div>` element that displays the active step caption and two [Button](/Documentation/Guide/UI_Components/Button/Overview/) components:

Reviewer 3: ```suggestion
To get started, add a Stepper and a MultiView to your project. Create a navigation panel - a container with child items. Child items are a `<div>` element that displays the active step caption and two [Buttons](/Documentation/Guide/UI_Components/Button/Overview/):
```

            <>
                <Stepper ... >
                    {steps.map((step) => <Item key={step.label} {...step} />)}
                </Stepper>
                <div className="content">
                    <MultiView ...
                        focusStateEnabled={false}
                        animationEnabled={false}
                        swipeEnabled={false}
                    >
                    </MultiView>
                    <div className="nav-panel">
                        <div className="current-step">
                            Step <span className="selected-index">1</span> of {steps.length}
                        </div>
                        <div className="nav-buttons">
                            <Button ...
                                id="prevButton"
                                text="Back"
                                visible={false}
                            >
                            <Button
                                id="nextButton"
                                text="Next"
                            />
                        </div>
                    </div>
                </div>
            </>
        )
    }

---

> The "Back" button is initially hidden. This tutorial implements logic that shows the button on the second step of the wizard later (see [Synchronize Steps and Navigation Panel](/Documentation/Guide/UI_Components/Stepper/Implement_a_Wizard/#Synchronize_Steps_and_Navigation_Panel)).

Reviewer: ```suggestion
The "Back" button is initially hidden. A later step in this tutorial implements dynamic visibility logic (see [Synchronize Steps and Navigation Panel](/Documentation/Guide/UI_Components/Stepper/Implement_a_Wizard/#Synchronize_Steps_and_Navigation_Panel)).
```
